packages/mds-ingest-service/repository/index.ts

```diff
@@ -0,0 +1,11 @@
+import { ReadWriteRepository } from '@mds-core/mds-repository'
+import entities from './entities'
+import migrations from './migrations'
+
+class IngestReadWriteRepository extends ReadWriteRepository {
+  constructor() {
+    super('devices', { entities, migrations })
```

Comment:
This will create a `devices-migrations` table. Should probably be `ingest` so you get `ingest-migrations`